### DISCUSSION TITLE
[5.x] FR: Option to Show Asset Thumbnail (Without Asset Title) in Element Index

### DISCUSSION THREAD

**Discussion by richhayler** [craftcms/cms#14813] [Ideas] [answered]
It would be great to have the option to show asset thumbnails without the asset title in element indexes within the control panel. 

Assets with long titles can unecessarily take up additional space and I would expect in most case 90% of the value would be displaying the asset thumbnail rather than the asset title.

![CleanShot 2024-04-15 at 18 00 44@2x](https://github.com/craftcms/cms/assets/2505643/ade374f5-a2ef-4019-b6cf-6934e69bf545)

**Comment by thupsi** [comment, marked as answer]
There is allready an option in the field settings (at the bottom) for this:
![image](https://github.com/craftcms/cms/assets/19857599/597b4151-44b6-414d-9260-a802cd114b6b)

**Reply by richhayler** [reply]
Awesome!  I completed missed this. Thank you @thupsi.
